ui/apps/moderation/src/pages/moderators.tsx

```diff
@@ -43,68 +29,46 @@ const AllModerators: React.FC<RootComponentProps> = () => {
     activeTab === 'All' ? moderator : moderator.status === activeTab.toLowerCase(),
   );
 
-  const handleTabClick = (value: string) => () => {
-    setActiveTab(value);
-  };
-
   const handleSocialLinkClick = () => {
     /** TODO: connect this */
   };
 
   return (
-    <VerticalFillBox gap="small">
-      <BasicCardBox>
-        <Box pad="medium" gap="small">
-          <Text size="xlarge" weight="bold">
-            {t('Moderators')}
-          </Text>
-          <Text color="subtitleText">
-            {t('Learn more about and reach out to your moderators.')}
-          </Text>
-        </Box>
-        <Box direction="row" width="98%" margin={{ vertical: '0', horizontal: 'auto' }}>
-          {modTabs.map((tab, idx) => (
-            <Box
-              key={tab.title + idx}
-              width="25%"
-              pad="small"
-              border={tab.value === activeTab ? { side: 'bottom', color: 'accentText' } : false}
-              onClick={handleTabClick(tab.value)}
-            >
-              <Text
-                size="large"
-                color={tab.value === activeTab ? 'accentText' : 'subtitleText'}
-                textAlign="center"
-              >
-                {tab.title}
-              </Text>
-            </Box>
-          ))}
-        </Box>
-      </BasicCardBox>
+    <BasicCardBox pad="p-0">
+      <ModerationSwitchCard tabs={modTabs} activeTab={activeTab} onTabClick={setActiveTab} />
 
-      {getModeratorsQuery.isFetching && <Spinner />}
+      {getModeratorsQuery.isFetching && (
+        <Box style="flex items-center justify-center p-4">
+          <Spinner size="lg" />
+        </Box>
+      )}
 
       {!getModeratorsQuery.isFetching &&
         getModeratorsQuery.data &&
         getModeratorsQuery.data.length > 0 && (
-          <ListWrapper>
-            <ListArea>
+          <Box style="flex-1">
```

Comment:
I think it's better to wrap the classes with tw or tx utilities just to make sure we don't have issues going further

Reply:
I changed the style prop on the other PR, so I will still update this when it's merged